apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/TransformerTestUtils.java

```diff
@@ -1444,6 +1444,8 @@ static void assertCommonEobInformationInpatientSNF(
         CcwCodebookVariable.CLM_DRG_CD,
         diagnosisRelatedGroupCd,
         eob.getDiagnosisFirstRep().getPackageCode());
+
+    assertEquals(1, eob.getDiagnosisFirstRep().getSequence());
```

Comment:
I would also assume we'd want to check this in the various claim type tests too.

Reply:
done its in the inpatient and snf claims

Reply:
Ah I see those tests call this common method where we do an assert.